common/src/main/java/net/whimxiqal/journey/data/migration/MysqlMigrationHandler.java

```diff
@@ -0,0 +1,40 @@
+package net.whimxiqal.journey.data.migration;
+
+import net.whimxiqal.journey.Journey;
+import net.whimxiqal.journey.data.DataVersion;
+import net.whimxiqal.journey.data.sql.SqlConnectionController;
+import net.whimxiqal.journey.data.sql.mysql.MySqlConnectionController;
+
+public class MysqlMigrationHandler extends SqlMigrationHandler {
+    public MysqlMigrationHandler() {
+        super(new MySqlConnectionController());
+    }
+
+    @Override
+    public void runMigration(DataVersion currentDataVersion) {
```

Comment:
Not a problem, but I was thinking of having this as a plural form `runMigrations` and internally it iterates in a loop with `while (version != DataVersion.latest())`, so the switch case it hit with every version between the current version and `latest()` (which, down the road, could be a bunch. Imagine someone with today's plugin (data `V000`) downloads another version of the plugin with data version `V005`. Then we'll have to run 4 migrations, so how is that repetition achieved?)

Although, it would work recursively too, and some migrations would just call `runMigration` again with the incremented version. So, recursively or iteratively, I don't particularly mind, but I just wanted to make sure that you knew about the plan to run multiple of these migrations in a row at some point.

Also, from an earlier comment, a return value of the incremented data version would be nice, because then if any migration has an error, the most up-to-date version can be returned (maybe `V003` or something from the previous example) and the `DataManager` would have an accurate version stored and can properly handle the case that the database is old